lib/src/lib/button/components/action-button-menu/lab900-action-button-menu.component.html

```diff
@@ -19,6 +21,8 @@
         mat-menu-item
         [matMenuTriggerFor]="innerMenu.actionMenu"
         [disabled]="getDisabled(action) | async"
+        [matTooltip]="action?.tooltip?.value | translate"
+        [matTooltipPosition]="action?.tooltip?.position ?? 'left'"
```

Comment:
Zou persoonlijk geen default voorzien, dat doet Angular.

Reply:
default is below, wat gaat overlappen met andere menu items, daarmee vond ik left beter
left overlapt ook niet met diepere menu items die naar rechts open gaan